### PR TITLE
refactor: reformat nuxt config 💄

### PR DESCRIPTION
**Pull request by pi0**
- Use one line comments 
- Use go.nuxtjs.dev shortlinks (this also allows us pointing users to updated link)
- Use `server: false` and target only if not defaults used
- One empty line between sections
- Improve descriptions
- Don't depend on `npm_package_`

**Rendered:** https://github.com/nuxt/create-nuxt-app/blob/d81a083d49e5fbec2c09e2c52d5590170a71c819/packages/create-nuxt-app/test/snapshots/index.test.js.md

**Current redirects:** https://github.com/nuxt/nuxt-redirects/blob/master/_redirects

### CODE REVIEW

**Review by clarkdo — APPROVED** [review]

File: packages/cna-template/template/nuxt/nuxt.config.js [hunk 1]
```diff
@@ -1,21 +1,16 @@
...
-  ** See https://nuxtjs.org/api/configuration-mode
-  */
-  mode: '<%= mode %>',
-  /*
-  ** Nuxt target
-  ** See https://nuxtjs.org/api/configuration-target
-  */
+<%_ } _%>export default {
+  <%_ if (mode === 'spa') { _%>
+  // Disable server-side rendering (https://go.nuxtjs.dev/config-mode)
```
Comment (pi0) [inline packages/cna-template/template/nuxt/nuxt.config.js:5]: We need a new link

Comment (Atinux) [reply packages/cna-template/template/nuxt/nuxt.config.js:5]: Like https://go.nuxtjs.dev/config-ssr ?

File: packages/cna-template/template/nuxt/nuxt.config.js [hunk 2]
```diff
@@ -1,21 +1,16 @@
...
+  server: false,
```
Comment (Atinux) [inline packages/cna-template/template/nuxt/nuxt.config.js:6]:
```suggestion
  ssr: false,
```

File: packages/cna-template/template/nuxt/nuxt.config.js (hunk 1 above)
Comment (pi0) [reply packages/cna-template/template/nuxt/nuxt.config.js:5]: Yes! Debbie just created page i think (not merged)

**Pull request merged**